styles.css

```diff
@@ -0,0 +1,195 @@
+    padding: 14px 16px;
+    text-decoration: none;
+    font-size: 17px;
+  }
+  #topnav a:hover {
+    background-color:  #CAE5FF;
+    color: #615D6C;
+  }
+
+  #topnav a.active {
+    background-color: lightskyblue;
+    color: white;
+  }
+
+  .home {
+    height: 80vh;
+    background-color: #615D6C;
+  }
+
+  .home_img {
+    height: 100%;
+    width: 100%;
+    object-fit: cover;
+  }
+
+  .about {
+    height: 80vh;
+    background-color: #615D6C;
+    color: #ACEDFF;
+  }
+
+  .about_img {
+    height: 30%;
+    width: 30%;
+    object-fit: cover;
+  }
+
+  .SOC {
+    background-color: #615d6c;
+    height: 100vh;
+    color: #ACEDFF;
+  }
+
+  .contact {
+    background-color: #615D6C;
+    color: #ACEDFF;
+    height: 100vh;
+  }
+
+  a.contact:visited {
+    color: #ACEDFF;
+  }
+
+  .logo {
+    object-fit: fill;
+    width: 100px;
+  }
+
+/* removes the underline under a link */
+  a:link {
+    text-decoration: none;
+  }
+/* the flex container becomes flexible by setting the display property to flex */
+  .flex-container {
+    /* display: flex;
+    flex-flow: row wrap;
+    height: 100vh;
+    justify-content: center;
+    align-items: center;
+    align-content: space-between; */
+    display: flex;
+    justify-content: space-around;
+  }
+
+  .flex-container > div {
+      background-color: #cae5ff;
+      /* width: 40vh;
+      height: 55vh;
+      margin: 10px;
+      padding: 20px;
+      font-size: 17px; */
+      width: 300px;
+      height: 400px;
+      margin: 30px;
+     box-shadow: 5px 5px 20px black ; 
+      padding: 10px;
+      color: #615d6c;
+  }
+
+  .flex-item > h3 {
+      text-align: center;
+  }
+
+  .github-octocat {
+      max-width: 100%;
+  }
+
+  .subtext {
+    text-align: center;
+    font-size: 15px;
+    font-weight: 300;
+  }
+
+  /* Contact Form Style */
+  * {
+    box-sizing: border-box;
+  }
+  /* Style text inputs, select elements and textareas */
+  input[type=text], select, textarea {
+    width: 100%;
+    padding: 12px;
+    border: 1px solid #ccc;
+    border-radius: 4px;
+    resize: vertical;
+  }
+    /* Style email inputs, select elements and textareas */
+  input[type=email], select, textarea {
+    width: 100%;
+    padding: 12px;
+    border: 1px solid #ccc;
+    border-radius: 4px;
+    resize: vertical;
+  }
+  /* Style the label to display next to the inputs */
+  label {
+    padding: 12px 12px 12px 0;
+    display: inline-block;
+    color: #615d6c;
+  }
+  /* Style the submit button */
+  input[type=submit] {
+    background-color: #6f8ab7;
+    color: white;
+    padding: 12px 20px;
+    border: none;
+    border-radius: 4px;
+    cursor: pointer;
+    float: right;
+  }
+  
+  input[type=submit]:hover {
+    background-color: #6f8ab7;
+  }
+  /* Style the container */
+  .contact-form-container {
+    border-radius: 5px;
+    background-color: #cae5ff;
+    padding: 20px;
+  }
+  /* Floating column for labels: 25% width */
+  .col-25 {
+    float: left;
```

Comment:
Float was used before CSS positions, flexbox and gridbox were created. Developers tend to avoid using float and instead use flexbox/gridbox instead as they are now heavily supported by browser types. Research this further to understand the reasons why and you can choose whether you want to update your code.

Reply:
Thank you Maxine, I'll do some research about float to understand it better and see if the flexbox can replace it.